optionals/bnae_trg42_compat_ace/CfgWeapons.hpp

```diff
@@ -0,0 +1,94 @@
+class Mode_SemiAuto;
+class asdg_OpticRail;
+
+class asdg_OpticRail1913: asdg_OpticRail {};
```

Comment:
Why do we change inheritance of that? It already inherits, just include it normally.